spec/lib/phobos/producer_spec.rb

```diff
@@ -151,7 +197,11 @@ class TestProducer1
       end
 
       describe 'with a delivery threshold set' do
-        let(:config_hash) { Phobos.config.producer_hash.merge(delivery_threshold: 10) }
+        let(:config_hash) do
+          hash = Phobos.config.producer_hash.merge(delivery_threshold: 10)
+          hash.delete(:persistent_connections)
```

Comment:
Is this still necessary now that we `.reject { |k, _| INTERNAL_PRODUCER_PARAMS.include?(k) }` ?

Reply:
ping: old comment